main.go

```diff
@@ -25,6 +27,25 @@ type CmdLineOptions struct {
 	Validate   bool   `long:"validate" description:"Validate configuration and exit"`
 }
 
+var (
+	BuildID string
+	VERSION string
+)
+
+func init() {
+	// set the version string to our desired format
+	if version.VERSION == "" {
```

Comment:
d'you mind adding a comment noting that `version.VERSION` is set via ldflags in `build.sh`?